### PR TITLE
Model card: CodeBERT fine-tuned for Insecure Code Detection

### PR DESCRIPTION
**Pull request by mrm8488**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to the it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

File: model_cards/mrm8488/codebert-base-finetuned-detect-insecure-code/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+language: en
```
Comment (julien-c) [inline model_cards/mrm8488/codebert-base-finetuned-detect-insecure-code/README.md:2]: datasets:
- codexglue

Comment (julien-c) [inline model_cards/mrm8488/codebert-base-finetuned-detect-insecure-code/README.md:2]:
```suggestion
language: en
datasets:
- codexglue
```

**Comment by julien-c** [comment]
You can add the dataset(s) id(s) even for datasets not currently implemented in the `datasets` lib. That way, it will prompt us, or someone from the community, to add it at some point :)

Actually, did you take a look at how to implement a new `dataset`, @mrm8488? We can help, cc @lhoestq @thomwolf

**Comment by mrm8488** [comment]
I didn't know I could add the dataset `id` if it was not available at HF/Datasets. I will do it next times. Thanks for letting me know @julien-c. And yes, I was talking with @thomwolf and I will try to add this dataset to HF/Datasets ASAP (this weekend).

**Pull request merged**